packages/client/src/schema.ts

```diff
@@ -33,7 +33,7 @@ export const Schemas = {
         .object({
           id: joi.string().uuid().required(),
           conversationId: joi.string().uuid().required(),
-          authorId: joi.string().uuid().required(),
+          authorId: joi.string().uuid().optional(),
           sentOn: joi.date().required(),
```

Comment:
The schema for messages was incorrect. A message can have a null author (bot message)